test/Bugs/randombug.baseline

```diff
@@ -63,10 +63,10 @@ Test case 30
 Passed
 Test case 31
 Error
-	at func2 (randombug.js:287:9)

```

Comment:
Fair enough. I didn't really intend to change this test, it was just part of a refactoring.